WTG.Analyzers/Analyzers/BooleanLiteral/ArgumentListExtensions.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace WTG.Analyzers
+{
+	static class ArgumentListExtensions
+	{
+		public static IParameterSymbol? TryFindCorrespondingParameterSymbol(this ArgumentListSyntax argumentList, int index, SemanticModel semanticModel, CancellationToken cancellationToken)
```

Comment:
Nit: I normally prefer to put context information (like SemanticModel) towards the start of the parameter list, so it feels odd having index before semanticModel.